libs/interpret/src/lib/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.tsx

```diff
@@ -134,23 +133,7 @@ export class GlobalExplanationTab extends React.PureComponent<
     const classNames = globalTabStyles();
 
     if (!this.props.jointDataset.hasLocalExplanations) {
-      if (this.props.globalImportance !== undefined) {
-        return (
-          <GlobalOnlyChart
-            metadata={this.props.metadata}
-            globalImportance={this.props.globalImportance}
-          />
-        );
-      }
-      return (
-        <div className={classNames.missingParametersPlaceholder}>
-          <div className={classNames.missingParametersPlaceholderSpacer}>
-            <Text variant="large" className={classNames.faintText}>
-              {localization.Interpret.GlobalTab.missingParameters}
-            </Text>
-          </div>
-        </div>
-      );
+      return <GlobalOnlyChart />;
```

Comment:
So for the global only chart, is it just showing top most important features for all data cohort, and it can't show feature importances for individual cohorts?  Interesting.

Reply:
right, we cannot filter data.